<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgWorkingTask</class>
 <widget class="QDialog" name="DlgWorkingTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>84</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Working</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="ActionLabel">
     <property name="text">
      <string notr="true">Working</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="ProgressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="CancelBt">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
